nix/automation/jobs.nix

```diff
@@ -2,16 +2,90 @@
   inputs,
   cell,
 }: let
-  inherit (inputs) nixpkgs;
+  inherit (inputs) nixpkgs iohkNix cardano-node;
+  inherit (nixpkgs) lib;
   inherit (inputs.cells.cardano) packages library nixosProfiles;
   inherit (inputs.bitte-cells._writers.library) writeShellApplication;
-  inherit (inputs.nixpkgs.lib.strings) fileContents;
+  inherit (lib.strings) fileContents;
+
+  cabal-project-utils = nixpkgs.callPackages iohkNix.utils.cabal-project { };
+
 in {
-  materialize-node = writeShellApplication {
-    name = "materialize-node";
+  update-mono-repo = writeShellApplication {
```

Comment:
you can use `meta.description` so that `std` shows a helpful description of these tasks